.github/workflows/python-wheels.yaml

```diff
@@ -107,9 +107,16 @@ jobs:
           fetch-tags: true
 
       - uses: actions/setup-python@v5
+        if: ${{ matrix.config.label != "windows" }}
         with:
           python-version: "3.12"
 
+      - uses: actions/setup-python@v5
+        if: ${{ matrix.config.label == "windows" }}
+        with:
+          python-version: "3.12"
+          architecture: x64
```

Comment:
Seems like setup-python still installs a 32 bit Python by default, while cibuildwheel is expected 64 bit on that host

Reply:
I believe so...I think cibuildwheel controls that, and since we don't override anything in the pyproject.toml I think it should still generate the same types of wheels as specified here:

https://cibuildwheel.pypa.io/en/stable/options/#build-skip

Admittedly my knowledge of Windows and how it builds for different architectures is limited, so definitely something to verify when wheels actually get build